src/main/java/microsoft/exchange/webservices/data/property/complex/FileAttachment.java

```diff
@@ -304,9 +310,11 @@ protected void handleStreamContentException(Exception e, File responseFile, Outp
     try {
       responseBytes = Files.readAllBytes(Paths.get(path));
 
-      // Print the response to the output file.
-      System.out.println("EWS response:");
-      System.out.println(new String(responseBytes, Charset.forName("UTF-8")));
+      if (ewsResponseTraceEnabled) {
+        // Print the response to the output file.
+        System.out.println("EWS response:");
+        System.out.println(new String(responseBytes, Charset.forName("UTF-8")));
+      }
```

Comment:
Now we'll only print the attachment contents to System.out if EwsResponse trace is enabled.